libs/components/molecules/src/lib/interactive-svg/interactive-svg.component.scss

```diff
@@ -17,4 +17,11 @@
     fill: hsl(0deg 100% 50%);
     mix-blend-mode: saturation;
   }
+
+  .hover-active & g > g:hover {
```

Comment:
Could you point me to the files which need this to work?

Reply:
It's really just the renal corpuscle svg. It has a weird format compared to the others so it will sometimes highlight only one cell instead of the entire group.

Reply:
Ok, I'll have a closer look at it

Reply:
Work with Supriya to get the SVG structured correctly. We shouldn't be building in hacks for individual SVGs, they should just be structured the way we need it. What can she change in the SVG that will alleviate this?